docs/source/templatesmod.rst

```diff
@@ -151,6 +151,14 @@ The following templates are bundled with SiteTree:
 
    Constructs nav-pills Bootstrap 3 horizontal navigation.
 
+ * `sitetree/menu_bootstrap3_deep.html`
+
+   Constructs Bootstrap 3 menu with infinite submenus.
+   Requires adding extra CSS::
+
+   <link href="/static/css/sitetree_bootstrap_submenu.css" type="text/css" rel="stylesheet" media="screen">
```

Comment:
``/static/`` shouldn't be harcoded, since it can change.

Reply:
Of course, that is mistake.